### PR TITLE
upgradation of angular 

### PR DESCRIPTION
**Pull request by moudgalya1223**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,26 +12,26 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^16.2.10",
+    "@angular/animations": "^18.2.7",
     "@angular/cdk": "^16.2.9",
```
Comment (whyvrafvr) [inline package.json:16]: Do not forget to update angular material as well...

File: package.json [hunk 2]
```diff
@@ -12,26 +12,26 @@
...
-    "@angular/common": "^16.2.10",
-    "@angular/compiler": "^16.2.10",
-    "@angular/core": "^16.2.10",
-    "@angular/forms": "^16.2.10",
+    "@angular/common": "^18.2.7",
+    "@angular/compiler": "^18.2.7",
+    "@angular/core": "^18.2.7",
+    "@angular/forms": "^18.2.7",
     "@angular/material": "^16.2.9",
```
Comment (whyvrafvr) [inline package.json:21]: Do not forget to update angular material as well...

**Review by whyvrafvr — CHANGES_REQUESTED** [review]
Check `Angular Material` as well...
> https://angular.dev/update-guide?v=16.0-17.0&l=1

and then
> https://angular.dev/update-guide?v=17.0-18.0&l=3

File: package.json (hunk 2 above)
Comment (moudgalya1223) [reply package.json:21]: hello @whyvrafvr  done .

File: package.json (hunk 1 above)
Comment (moudgalya1223) [reply package.json:16]: created a pr for this issue

**Comment by whyvrafvr** [comment]
fixed

**Pull request closed without merging**